pkg/console/install_panels.go

```diff
@@ -1045,6 +1060,44 @@ func addNetworkPanel(c *Console) error {
 	setLocation(askInterfaceV.Panel, 3)
 	c.AddElement(askInterfacePanel, askInterfaceV)
 
+	// askVlanIDV
+	askVlanIDV.PreShow = func() error {
+		if mgmtNetwork.VlanID != 0 {
+			askVlanIDV.Value = strconv.Itoa(mgmtNetwork.VlanID)
+		}
+		return nil
+	}
+	validateVlanID := func() (string, error) {
+		vlanIDStr, err := askVlanIDV.GetData()
+		if err != nil {
+			return "", err
+		}
+		if vlanIDStr == "" {
+			mgmtNetwork.VlanID = 0
+			return "", nil
+		}
+		var vlanID int
+		vlanID, err = strconv.Atoi(vlanIDStr)
+		if err != nil {
+			return ErrMsgVLANShouldBeANumberInRange, err
+		}
+		// 0 is unset
+		if vlanID < 0 || vlanID > 4094 {
+			return ErrMsgVLANShouldBeANumberInRange, fmt.Errorf(ErrMsgVLANShouldBeANumberInRange)
+		}
+		mgmtNetwork.VlanID = vlanID
```

Comment:
By typing an invalid value, user can't see the error.
The installer simply restarts.

Reply:
fixed